app/models/user.rb

```diff
@@ -0,0 +1,11 @@
+class User < ApplicationRecord
+	before_save { self.email = email.downcase }
```

Comment:
- indent cách 2 spaces đầu dòng thay vì 4
```suggestion
	before_save {email.downcase!}
```